packages/admin/admin/src/hooks/useAsyncOptionsProps.ts

```diff
@@ -4,24 +4,34 @@ export interface AsyncOptionsProps<T> {
     isAsync: boolean;
     open: boolean;
     options: T[];
+    error: boolean;
```

Comment:
I'd prefer exposing the actual error instead of a boolean. For instance, this could be used for error reporting via Sentry.

Reply:
Had this already implemented locally, but due to MUI Select component has typed error as boolean, I decided to reuse this prop.

Changed this now, and forwarded the error prop as `Error | null` to the FinalFormSelect and handling it there accordingly.

Reply:
I wasn't aware that MUI has an error prop already. So maybe it's okay to use the boolean approach after all. Sorry 😬

Reply:
due to: https://github.com/vivid-planet/comet/pull/3863#discussion_r2101810722 i kept the type Error but renamed the prop to `loadingError` - made more sense to me.